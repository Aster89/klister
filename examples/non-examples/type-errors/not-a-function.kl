#lang kernel

(example (42 23))
