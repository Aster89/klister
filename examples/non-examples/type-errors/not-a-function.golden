Type mismatch at not-a-function.kl:3.11-3.13.
Expected (Integer â†’ ?1) but got Integer
